client/src/components/VtkViewer.vue

```diff
@@ -85,12 +104,31 @@ export default {
   mounted() {
     this.initializeView();
     this.initializeSlice();
+    this.updateCrosshairs();
+    this.renderSubscription = this.view.getInteractor().onRenderEvent(() => {
+      this.updateCrosshairs();
+    });
+    this.resizeObserver = new window.ResizeObserver((entries) => {
+      if (entries.length === 1) {
```

Comment:
What are the `entries` here?

Reply:
Haha I had to ask Forrest the same thing. Part of the `ResizeObserver` [API](https://developer.mozilla.org/en-US/docs/Web/API/Resize_Observer_API) explains that the observer can observe multiple elements that got resized with the same action. This line was his recommendation.

Reply:
To make sure the logic matches the intent... if we receive one of these events that has multiple entries, we intentionally want to ignore it?

Reply:
Yes, we tell the observer to only observe one element per instance of the `VtikViewer` component. We don't want to execute this block for any entries other than the specified target `this.$refs.viewer`.
We have three instances of the `VtkViewer` component; without this `if`, this block would execute 3^2 times instead of 3 times.